src/udev/udevadm-trigger.c

```diff
@@ -41,13 +41,33 @@ static int exec_list(sd_device_enumerator *e, const char *action, Set **settle_s
 
                 r = sd_device_set_sysattr_value(d, "uevent", action);
                 if (r < 0) {
+                        /* ENOENT may be returned when a device does not have /uevent or is already
+                         * removed. Hence, this is logged at debug level and ignored.
+                         *
+                         * ENODEV may be returned by some buggy(?) devices e.g. /sys/devices/vio.
+                         * See, https://github.com/systemd/systemd/issues/13652#issuecomment-535129791
+                         * and https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1845319.
+                         * So, this error is also logged at debug level, and ignored.
```

Comment:
should be LOG_WARN i think. We want people to fix this, hence the message should be seen